integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -3090,7 +3086,7 @@ msgstr "Bei Landkreisen geben Sie die Postleitzahl des Verwaltungssitzes an."
 
 #: cms/models/regions/region.py:187
 msgid "email address of the administrator"
-msgstr "E-Mail-Adresse des Administrators"
+msgstr "E-Mail-Adresse des*r Administrator*in"
```

Comment:
```suggestion
msgstr "E-Mail-Adresse der Verwaltung"
```
I think `des*r` is invalid German - it would have to be `des*der` which we decided to avoid...

Reply:
(Rejected) After our talk with @osmers we landed on "%(BRANDING)-Teammitglied"